docs/source/single_page_run_through.rst

```diff
@@ -302,40 +303,37 @@ choosing, without having to re-write the entire newton_raphson wrapper function.
 In `fastats` this is performed by changing the semantics of positional and keyword
 arguments; `numba` does not allow us to pass functions as arguments, but even if it
 did, we still need the ability to arbitrarily modify deeply nested functions (use
-cases discussed below).
+cases discussed below), rather than just the calls in the top-level function.
 
 The Newton-Raphson code shown above is in the `fastats` standard library, and allows us
 to do the following::
 
     from fastats.optimise.root_finding.newton_raphson import newton_raphson
+    from pytest import approx
 
-    def my_func(x, y):
-        return x**2 + y**2
+    def my_func(x):
+        return x ** 3 - x - 1
 
-    my_solver = newton_raphson(, 0.001, root=my_func)
+    result = newton_raphson(0.5, 0.001, root=my_func)
+    assert result == approx(1.324717)
```

Comment:
Out of scope: need to enable pytest in docs.

Reply:
Nice one, thanks. Updated to use double backticks as well